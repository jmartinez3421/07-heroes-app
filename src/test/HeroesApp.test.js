import { shallow } from 'enzyme';
import React from 'react';
import HeroesApp from '../HeroesApp';

describe('Pruebas sobre el componente <HeroesApp />', () => {
    test('Debe mostrar el componente correctamente', () => {
        const wrapper = shallow(<HeroesApp />);

        expect(wrapper).toMatchSnapshot();
    })
    
})