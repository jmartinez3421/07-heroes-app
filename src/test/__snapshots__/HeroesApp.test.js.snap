// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre el componente <HeroesApp /> Debe mostrar el componente correctamente 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
